src/helm/proxy/models.py

```diff
@@ -374,6 +374,31 @@ def engine(self) -> str:
         "JavaScript subset of The Stack (v1.1).",
         tags=[CODE_MODEL_TAG],
     ),
+    # Local HuggingFace models
+    Model(
+        group="local",
+        creator_organization="Meta",
+        name="local/llama-7b",
```

Comment:
I think these should all be `group="huggingface"` and `name="huggingface/llama-7b"` for consistency.

I would like this to be `meta/llama-7b` but unfortunately the pre-existing code base ties the group to the "client" used to serve the model, which is why we have `huggingface/gpt2` instead of `openai/gpt2` or `together/t5-11b` instead of `google/t5-11b`. I would like to make everything be the correct group eventually, but for now we should be consistent with existing conventions. It's also not obvious to me that "local" means "local Hugging Face" as opposed to "local some other framework".

Also unfortunately these model names are annoying to change because we have to do a MongoDB migration every time we change one of these.

cc @percyliang for opinions about naming